ApollosProject/apollos-apps
# fix(prayer): trigger pray and navigate to prayer submit after last pr…

## ✏️ Description
- trigger pray and navigate to prayer submit after last prayer



## 🔬 To Test
1. Open up the Prayer Experience
2. Pray/scroll through ALL prayers in the list until you get to the very last one
3. Press the "pray" button and you should continue to the prayer submit screen
4. 


## 📝 Checklist
